usps-processor@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=USPS Processor
+
+After=docker.service
+Requires=docker.service
+After=consul@%i.service
+Wants=consul@%i.service
+After=wildfly@%i.service
+Requires=wildfly@%i.service
+
+[Service]
+EnvironmentFile=/etc/environment
+TimeoutStartSec=10min
+TimeoutStopSec=10min
+
+Environment=DOCKER_REPO=quay.io/democracyworks/usps-processor
+Environment=VERSION=buildkite-coreos-fd55ab2
+Environment=CONTAINER=usps-processor
+Environment=HOME=/root
+
+ExecStartPre=-/usr/bin/docker kill ${CONTAINER}
+ExecStartPre=-/usr/bin/docker rm ${CONTAINER}
+ExecStartPre=/usr/bin/curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/buildkite/dockercfg?raw -o /root/.dockercfg
```

Comment:
This will need a sleep before curling from Consul. I've started w/ `sleep 2` but haven't had a chance to test that much. This mostly comes up when a node is rebooted and everything gets started up all at once.